src/translations/LanguageContext.tsx

```diff
@@ -36,9 +36,13 @@ function useLanguage() {
 
   useEffect(() => {
     if (useSystemLanguage) {
+      // Get preferred locale from system settings.
+      // If no system settings match our languages we select
+      // English as it is more likely they know english than Norwegian.
+      // If locale is undefined we don't support any of the system languages.
       const language = locale
         ? getAsAppLanguage(locale.languageCode)
```

Comment:
I kinda feel some of the logic in this file would be easier to follow if `locale` was more strongly typed. As far as I understand it will always be either Norwegian or English, and never undefined.

Reply:
My bad. I see now that it can be undefined.